features/mbedtls/targets/TARGET_STM/aes_alt.h

```diff
@@ -30,6 +30,8 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+#define ST_AES_TIMEOUT    ((uint32_t) 3)
```

Comment:
Could you please add a comment explaining the rationale behind this #define? Also, it would be good to document what units are used and why is 3 a good choice.

Reply:
I must check that the crypto processor is not busy before reading / writing some registers. 
I cannot use a while loop without any timeout. 
The minimum timeout is 1 ms, which is more than sufficient for the crypto processor to process any command.
I change this value to 1 and add a comment ` /* 1 ms timeout is sufficient for the crypto processor */`

Reply:
I think that instead of lowering the timeout to one it would be better to raise it to a very high value, preferably high enough to make the behaviour practically synchronous, because:
- it has no performance toll when the accelerator is working properly
- the interface of mbed TLS crypto modules is inherently synchronous and applications (with the TLS stack in mbed TLS among them) are not prepared for asynchronous use and low timeout here would result in highly unreliable behaviour.

Reply:
How many milliseconds would you consider as high enough ? The default value used in ST examples codes is 0xFF

Reply:
Considering that as you say 1 ms should be fine in most cases, 0xFF seems good enough to me.